client/me/help/help-contact/index.jsx

```diff
@@ -239,19 +228,13 @@ class HelpContact extends React.Component {
 	};
 
 	shouldUseHappychat = () => {
-		const { olark } = this.state;
-
 		// if happychat is disabled in the config, do not use it
 		if ( ! config.isEnabled( 'happychat' ) ) {
 			return false;
 		}
 
 		// if the happychat connection is able to accept chats, use it
-		return (
-			this.props.isHappychatAvailable &&
-			olark.isUserEligible &&
-			this.props.isSelectedHelpSiteOnPaidPlan
-		);
+		return this.props.isHappychatAvailable && this.props.isSelectedHelpSiteOnPaidPlan;
```

Comment:
This will need the `isUserEligibleForHappychat` selector when the changes in `isHappychatAvailable` are reverted.